crates/apollo-mcp-server/src/operations.rs

```diff
@@ -433,6 +433,67 @@ pub fn operation_defs(
     Ok(Some((document, operation, comments.map(|c| c.to_string()))))
 }
 
+pub fn extract_and_format_overridden_variable_description(string: &str) -> Option<String> {
```

Comment:
I think we can reference the operation description cleanup in `fn tool_description(`
So this function would essentially become this
```
comments.and_then(|comments| {
            let content = Regex::new(r"(\n|^)\s*#")
                .ok()?
                .replace_all(comments.as_str(), "$1");
            let trimmed = content.trim();

            if trimmed.is_empty() {
                None
            } else {
                Some(trimmed.to_string())
            }
        })
```
And be used in both spots.